.github/workflows/check-release.yml

```diff
@@ -0,0 +1,11 @@
+on: [push, pull_request]
+name: Check & Release
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: lint
+        uses: Roang-zero1/factorio-mod-luacheck@master
```

Comment:
I'd rather not rely on some other person maintaining a thing.

On travis, installing luacheck is [a few lines of shell](https://github.com/minetest/minetest_game/blob/master/.travis.yml), can Github Actions do this too?

Reply:
Sure; fixed.